DSCResources/MSFT_xDnsServerADZone/MSFT_xDnsServerADZone.schema.mof

```diff
@@ -8,4 +8,5 @@ class MSFT_xDnsServerADZone : OMI_BaseResource
     [Write, Description("DNS Server name")] String ComputerName;
     [Write, Description("Credential used to set zone"), EmbeddedInstance("MSFT_Credential")] String Credential;
     [Write, Description("Whether the DNS zone should be available or removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Read] String CimSession;
```

Comment:
Okay, well for the unit test; none of the current resources in this module validate the Hashtable items. This is also kind of difficult since not always, all items are returned. For instance, In this resource 'MSFT_xDnsServerADZone' when calling the Get-TargetResource on a DNS zone that does not exist, the 'Name' item in the Hashtable is empty and validation would return a 'null-valued expression' error.

Furthermore, not all items mentioned in the schema are returned. For instance 'ComputerName' and 'Credential'. If I would create a unit test, than I think all 8 items from the schema should be populated to be consistent, what do you think? 
At least the HashTable should return type correct items so I could test in the Unit test if a valid type is returned. 

Anyway, it involves a little more change than only adding the Read property to the schema. But I'm willing to make the modification, but would like to know what do you think. Also because the other resources do not contain unit tests that validate the specific Hashtable items. I think that the other resources should be changed accordingly.
Please let me know what you think about it.

I'll change the Readme when I know which modifications I need to do. (If necessary)